kubemarine/plugins/__init__.py

```diff
@@ -135,7 +135,8 @@ def install(cluster, plugins=None):
     plugins_queue = []
     max_priority = 0
     for plugin_name, plugin_item in plugins.items():
-        if plugin_item.get("install", False) and plugin_item.get("installation", {}).get('procedures') is not None:
+        if utils.true_or_false(plugin_item.get("install", False)) == 'true' and \
```

Comment:
Yet another places in
* utils.make_ansible_inventory
* calico.py enrich_inventory
* few places in check_paas

Probably also add method like plugins.is_to_be_installed(inventory, plugin_name)

Reply:
`plugins.is_to_be_installed(inventory, plugin_name)` is not so good, because during concertation we can give 3 variants: true, false and unsupported (value that can't be converted).  
Missed places were fixed

Reply:
> we can give 3 variants: true, false and unsupported (value that can't be converted).

and actually this is a good point. In different places of code we consider "undefined" as False (for example, in plugins.install) and as True (for example in nginx_ingress.py) even for the same plugin. I suggest to throw exception for "undefined" during enrichment. (plugins.verify_inventory or better in some new procedure which will be run before any plugin enrichment). After this, it would be still possible to introduce some new method like is_to_be_installed